<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerminalSettingDialog</class>
 <widget class="QDialog" name="TerminalSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>180</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Terminal Settings</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QHBoxLayout" name="Layout1">
     <item>
      <widget class="QLabel" name="lblRouterAddr">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Router Address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtRouterAddr"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Layout2">
     <item>
      <widget class="QLabel" name="lblPassword">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPassword"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Layout3">
     <item>
      <widget class="QLabel" name="lblUsername">
       <property name="minimumSize">
        <size>
         <width>105</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>105</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtUsername"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="Spacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="Spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDefault">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TerminalSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TerminalSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
